framework/src/actions/CommonOutputAction.C

```diff
@@ -224,6 +230,14 @@ CommonOutputAction::act()
          _app.getParam<bool>("timing")))
       create("PerfGraphOutput");
 
+    if (!_app.getParam<bool>("no_timing") && getParam<bool>("perf_graph_live"))
+      _perf_graph.setLivePrintActive(true);
+    else
+      _perf_graph.setLivePrintActive(false);
+
+    _perf_graph.setLiveTimeLimit(getParam<Real>("perf_graph_live_time_limit"));
+    _perf_graph.setLiveMemoryLimit(getParam<unsigned int>("perf_graph_live_mem_limit"));
```

Comment:
```suggestion
    _perf_graph.setLivePrintActive(!_app.getParam<bool>("no_timing") && getParam<bool>("perf_graph_live"));
    _perf_graph.setLiveTimeLimit(getParam<Real>("perf_graph_live_time_limit"));
    _perf_graph.setLiveMemoryLimit(getParam<unsigned int>("perf_graph_live_mem_limit"));
```

Reply:
It's a coin toss on this - but I think I would rather see the logic more clearly written out.